docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+# Configuration for the services that are required to run an analysis of the
+# data from riff edu mattermost.
+# see https://www.metabase.com/docs/latest/operations-guide/running-metabase-on-docker.html
+version: '3.6'
+services:
+
+  target-db:
```

Comment:
I've found that good service names have a short prefix (e.g. `pfm-`, `edu-`) and then a relatively short descriptive name, so perhaps for this service something like `ba-mmdb` or something else that makes sense to you.

Reply:
Okay makes sense, I've added consistent names to the services.